app/views/mail_shared/_outlook_fix.html.erb

```diff
@@ -0,0 +1,1136 @@
+<style>
+img.no-float-image{
```

Comment:
You also probably don't need to qualify that selector with the `img` tag. You should just try and use pure classes.